AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+string n = Console.ReadLine();
+string mystring = Console.ReadLine();
+
+int count = 0;
+
+for(int i = 0; i < mystring.Length - 1; i++)
```

Comment:
### Alternative way. 
 - mystring.Length - 1 can replaced by n - 1 (should convert n as integer)

**Just for hint**